testing/core/tests/BUILD.bazel

```diff
@@ -57,9 +44,9 @@ expand_location_unit_test_suite()
         srcs = ["test_output.sh"],
         args = [
             "2",
-            "$(locations @output-filegroup-test//:include)",
+            "$(locations {})".format(nix_pkg("rules_nixpkgs_core_testing", "output-filegroup-test", "//:include")),
         ],
-        data = ["@output-filegroup-test//:include"],
+        data = [nix_pkg("rules_nixpkgs_core_testing", "output-filegroup-test", "//:include")],
```

Comment:
I was a bit confused why the `nix_pkg` module extension was being used like this, but now I see `nix_pkg` here is a helper macro from `defs.bzl`.

Reply:
Probably something to make clear in the docs. Or, if we end up getting `local_use_repo` the problem would go away.